app/web/features/profile/view/leaveReference/LeaveReferencePage.tsx

```diff
@@ -76,15 +78,15 @@ export default function LeaveReferencePage() {
           user.friends === User.FriendshipStatus.FRIENDS) ||
         (hostRequestId &&
           availableRefrences.availableWriteReferencesList.find(
```

Comment:
```suggestion
          availableReferences.availableWriteReferencesList.find(
```
And on line 73 (can't highlight because of no change)